.github/workflows/test.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['master']
+
+jobs:
+  build:
+    name: Build and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java:
+          - adopt@1.8
+          - graalvm-ce-java11@20.3.0
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Restore cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: cache-v1-${{ hashFiles('build.sbt') }}
+
+      #- name: Compile and Test
+      # run: ./sbt +test
+
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+
+      - name: Setup Jekyll
+        run: gem install jekyll -v 4.0.0
+
+      - name: Build the microsite
+        run: ./sbt microsite/makeMicrosite
+
+      - name: Upload site data
+        uses: actions/upload-artifact@v2
+        with:
+          name: site
+          path: site/target/site
+          retention-days: 1
```

Comment:
Since this _" artefact"_ is only used for sharing data between the two jobs, I believe it makes sense to only retain it for one day instead of for `90` which is the default.